### DISCUSSION TITLE
How can I check all the registered users in vaultwarden?

### DISCUSSION THREAD

**Discussion by cukewu** [dani-garcia/vaultwarden#5051] [Q&A]
I want to know how many users are registered, but it seems that the web page can only show the currently logged in users. Do I need to enter the database?

**Comment by BlackDex** [comment]
Not sure where you are looking. 
Butt via the Vaultwarden Admin Interface `/admin` you can see all users and organizations.

**Reply by cukewu** [reply]
The page prompts The admin panel is disabled, please configure the 'ADMIN_TOKEN' variable to enable it

**Reply by BlackDex** [reply]
https://github.com/dani-garcia/vaultwarden/wiki/Enabling-admin-page
